src/electionguard_cli/steps/e2e/submit_votes_step.py

```diff
@@ -15,11 +15,15 @@
     ElectionFactory,
 )
 
-from ..cli_models import E2eInputs, BuildElectionResults, E2eSubmitResults
-from .e2e_step_base import E2eStepBase
+from electionguard_cli.cli_models.e2e.e2e_inputs import E2eInputs
```

Comment:
use relative import

Reply:
I'm happy to switch it to a module import, but I've been having two problems with relative imports:

1. Do you ever get these linting errors in Visual Studio when you attempt to use relative imports that go up a directory?

![image](https://user-images.githubusercontent.com/1769905/164722538-22ba0976-0d82-4584-b6bc-a53e067f2b79.png)

If so, what do you do about them?

2. When I `make mypy` with any `from ...` commands I get the following linting errors:

`/steps/e2e/submit_votes_step.py:18:1: error: Relative import climbs too many namespaces`

Should I look into disabling that mypy linting error?

Reply:
I was able to solve issue #1 by deleting `/electionguard_cli/__init__.py`.  That also solved another linting problem I was having.  Still need guidance on issue #2.